src/test/resources/FindUser.feature

```diff
@@ -0,0 +1,31 @@
+Feature:
+  As a user
+  I want to be find another user
```

Comment:
? xd. Is this english?

Reply:
STRG+C/STRG+P sometimes is an ... 
But maybe... maybe i should be more attentive ... maybe